RNTester/e2e/__tests__/InputAccessoryView-test.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails oncall+react_native
+ * @format
+ */
+
+/* global device, element, by, expect */
+const {openComponentWithLabel} = require('../e2e-helpers');
+
+describe('InputAccessoryView', () => {
+  beforeAll(async () => {
+    await device.reloadReactNative();
+    await openComponentWithLabel(
+      '<InputAccessoryView>',
+      '<InputAccessoryView> Example showing how to use an InputAccessoryView to build an iMessage-like sticky text input',
+    );
+  });
+
+  it('Send button should show alert on press', async () => {
```

Comment:
Ideally,  the test title should read like a sentence: _"it should show alert on press"_, which in the test runner expands to _InputAccessoryView should show alert on press_.

However, that is not an appropriate description of our test case, because the InputAccessoryView only handles the positioning of the element, and not the click of a button.

Ideally, a test for InputAccessoryView would check that the component moves as keyboard is moved, but as discussed, that's currently not supported by Detox. So instead, maybe a better test case here is to say, _InputAccessoryView should be visible_ (where we test that the element is visible on screen), and _InputAccessoryView should be interactable_, where we test that the button is pressed as tested below?

There's many ways to do this, none idea, but that's what I would do.